fastlane/Fastfile

```diff
@@ -115,29 +115,31 @@ platform :ios do
     push_pods
     carthage_archive
     github_release(version: version_number)
-    prepare_next_version(version_number)
```

Comment:
this file is probably easier to read side-by-side. This separates the `prepare_next_version` into a new lane, so that it runs from a clean git checkout